assignment 1/flatten.js

```diff
@@ -0,0 +1,14 @@
+// Problem 3: Complete the flatten function that takes a JS Object, returns a JS Object in flatten format (compressed)
+function flatten(unflatObject) {
+  var output = {};
+  for (var iteration in unflatObject) {
```

Comment:
@abansal26 this is hilarious! It seems there are only 2 names for a variable used in iteration. Either index or iteration.
We suggested to name variables properly with @sakodiya giving an example to change to index, you changed all to index, to your luck it was relevant in all places but this. I comment you to change it and you revert back to iteration. :D
Don't be a doer be an engineer, read carefully, understand what the comment means, think whether your fix actually resolves that comment?

Reply:
I now understand what he meant. In this case as we are traversing through object i am now using property as variable to traverse through it. Commit ID - db0d8f6dc6080ddc97bc8d2dc7547bef4105a76f